doc/api/cli.md

```diff
@@ -6,6 +6,9 @@
 Node.js comes with a variety of CLI options. These options expose built-in
 debugging, multiple ways to execute scripts, and other helpful runtime options.
 
+Worker threads inherit non-process-specific options by default. Refer to
+[`Worker constructor options`][] to know how to customize worker thread options.
```

Comment:
Would it be better to specifically mention the `argv` and `execArgv` options? Otherwise, the user may have to do a bit of looking for a needle in a haystack in the worker threads doc.

Reply:
@Trott, thanks. Mentioned `argv` and `execArgv`.